app/components/media/wrapper_component.rb

```diff
@@ -2,11 +2,31 @@
 
 module Media
   class WrapperComponent < ViewComponent::Base
-    def initialize(file:, type:, resource_index:, thumbnail:)
+    def initialize(file:, type:, resource_index:, thumbnail:, size:)
       @file = file
       @type = type
       @resource_index = resource_index
       @thumbnail = thumbnail
+      @size = size
+    end
+
+
+    def prev_next_container
+      tag.div class: 'prev-next-container w-100' do
+        tag.div(class: 'prev-next-buttons') do
+          tag.button(arrow, class: 'left-arrow', disabled: @resource_index == 0, :onclick => "document.querySelector(`[data-content-list-index-param='#{@resource_index-1}']`).click()") +
```

Comment:
I don't totally love this solution, but I also don't love the idea of creating a full stimulus controller to click a button or emit an event.

Reply:
I did try to do this instead of onclick: data: {"action"=>"click->content-list#showMedia", "content-list-index-param"=> @resource_index + 1} but it caused problems with the showMedia JS and would require a rewrite of the code.